Client/src/Components/Sidebar/index.jsx

```diff
@@ -29,7 +29,7 @@ import LogoutSvg from "../../assets/icons/logout.svg?react";
 import Support from "../../assets/icons/support.svg?react";
 import Dashboard from "../../assets/icons/dashboard.svg?react";
 import Account from "../../assets/icons/user-edit.svg?react";
-import StatusPages from "../../assets/icons/status-pages.svg?react";
+/* import StatusPages from "../../assets/icons/status-pages.svg?react"; */
```

Comment:
If the commented out imports aren't needed anymore we can remove them

Reply:
All right!

Reply:
Done on this commit https://github.com/bluewave-labs/bluewave-cae03c7c9870aace797e6f1c11a3709382e70cab